osext/register.go

```diff
@@ -36,15 +36,21 @@ func ForceProcessShutdown(ctx context.Context) {
 	rID := GetRunID(ctx)
 
 	for _, pid := range processRegister[rID] {
-		p, err := os.FindProcess(pid)
-		if err != nil {
-			// optimistically continue and don't kill the process
-			continue
-		}
-		// no need to check the error for waiting the process to release
-		// its resources or whether we could kill it as we're already
-		// dying.
-		_ = p.Release()
-		_ = p.Kill()
+		Kill(pid)
 	}
 }
+
+// Kill will look for and kill the process with the
```

Comment:
This change will also mean we shouldn't need the `runID` work that is in https://github.com/grafana/xk6-browser/pull/755.